cairo_client/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "webauthn-sdk"
+name = "cairo-client"
```

Comment:
what is this crate for?

Reply:
I'm going to rename the crate in the next PR
It will be much easier